contracts/domain/BosonConstants.sol

```diff
@@ -31,6 +31,7 @@ string constant ALREADY_INITIALIZED = "Already initialized";
 string constant PROTOCOL_INITIALIZATION_FAILED = "Protocol initialization failed";
 string constant VERSION_MUST_BE_SET = "Version cannot be empty";
 string constant ADDRESSES_AND_CALLDATA_LENGTH_MISMATCH = "Addresses and calldata must be same length";
+string constant WRONG_CURRENT_VERSION = "Wrong current protocol versions";
```

Comment:
:( sorry, it should be version and not versions

Reply:
fixed